src/app/models/account.ts

```diff
@@ -3,4 +3,5 @@ import {AccountType} from './AccountType';
 export interface Account {
   accountName: string;
   type: AccountType;
+  user?: string
```

Comment:
This is not necessary, where you use it in `editPlainAccountToWorkSpace ` it's sufficient to call `session.account as AwsPlainAccount` so you get autocompletion for the Plain account properties, including User.